File: src/Persona.java
// Súper Clase - Clase Padre
public class Persona {
    public String nombre;
    public String apellido;
    public String direccion;

    public Persona(String nombre, String apellido, String direccion) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.direccion = direccion;
    }
    public String mostrarInfo(){
        return String.format("Nombre: %s, Apellido: %s, Direccion: %s",nombre,apellido,direccion);

    }
}

File: src/Persona.py
class Persona:
    def __init__(self, nombre, apellido, direccion):
        self.__nombre = nombre
        self.__apellido = apellido
        self.__direccion = direccion

    def mostrar_info(self):
        return f"Nombre: {self.__nombre}, Apellido: {self.__apellido}, Direccion: {self.__direccion}"
